src/test/java/easytests/support/QuestionTypesSupport.java

```diff
@@ -0,0 +1,135 @@
+package easytests.support;
+
+import easytests.core.entities.QuestionTypeEntity;
+import easytests.core.models.QuestionTypeModelInterface;
+import org.junit.Assert;
+import org.mockito.Mockito;
+
+import java.util.List;
+
+
+/**
+ * @author Vlasovigor
+ */
+public class QuestionTypesSupport {
+
+    protected static Object[][] fixtures = new Object[][]{
+            {
+                    1,
+                    "Один ответ",
+            },
+            {
+                    2,
+                    "Много ответов",
+            },
+            {
+                    3,
+                    "Нумерация",
+            },
+            {
+                    4,
+                    "Текст",
+            },
+    };
+
+    public QuestionTypeEntity getEntityFixtureMock(Integer index) {
+        return this.getEntityMock(fixtures[index]);
+    }
+
+    private QuestionTypeEntity getEntityMock(Object[] data) {
+        return this.getEntityMock(
+                (Integer) data[0],
+                (String) data[1]
+        );
+    }
+
+    private QuestionTypeEntity getEntityMock(
+            Integer id,
+            String name
+    ) {
+        final QuestionTypeEntity question_typeEntity = Mockito.mock(QuestionTypeEntity.class);
+        Mockito.when(question_typeEntity.getId()).thenReturn(id);
+        Mockito.when(question_typeEntity.getName()).thenReturn(name);
+        return question_typeEntity;
+    }
+
+    public QuestionTypeModelInterface getModelFixtureMock(Integer index) {
+        return this.getModelMock(fixtures[index]);
+    }
+
+    private QuestionTypeModelInterface getModelMock(Object[] data) {
+        return this.getModelMock(
+                (Integer) data[0],
+                (String) data[1]
+        );
+    }
+
+    private QuestionTypeModelInterface getModelMock(
+            Integer id,
+            String name
+    ) {
+        final QuestionTypeModelInterface question_typeModel = Mockito.mock(QuestionTypeModelInterface.class);
+        Mockito.when(question_typeModel.getId()).thenReturn(id);
+        Mockito.when(question_typeModel.getName()).thenReturn(name);
+        return question_typeModel;
+    }
+
+    public void assertEquals(QuestionTypeEntity expected, QuestionTypeEntity actual){assertEquals(expected, actual, false);}
```

Comment:
make multiple lines, do not write methods in one line.
fix all another methods like that one.